Python/ceval.c

```diff
@@ -4309,6 +4313,41 @@ _PyEval_EvalFrameDefault(PyThreadState *tstate, InterpreterFrame *frame, int thr
             DISPATCH();
         }
 
+        TARGET(LOAD_METHOD_MODULE): {
+            assert(cframe.use_tracing == 0);
+            LOAD_MODULE_ATTR_OR_METHOD(LOAD_METHOD);
+            Py_INCREF(res);
+            SET_TOP(NULL);
+            Py_DECREF(owner);
+            PUSH(res);
+            DISPATCH();
+        }
+
+        TARGET(LOAD_METHOD_CLASS): {
+            /* LOAD_METHOD, for class methods */
+            assert(cframe.use_tracing == 0);
+            SpecializedCacheEntry *caches = GET_CACHE();
+            _PyAdaptiveEntry *cache0 = &caches[0].adaptive;
+            _PyAttrCache *cache1 = &caches[-1].attr;
+            _PyObjectCache *cache2 = &caches[-2].obj;
+            _PyObjectCache *cache3 = &caches[-3].obj;
+
+            DEOPT_IF(TOP() != cache3->obj, LOAD_METHOD);
```

Comment:
Why? The version tag should be sufficient.

Reply:
Wait, facepalm on my part 😆 . I forgot that all types must have a ``__dict__`` of some sort.